python/trifinger_simulation/visual_objects.py

```diff
@@ -75,40 +76,40 @@ def set_state(self, positions):
             )
 
 
-class CubeMarker:
-    """Visualize a cube."""
-
+class ObjectMarker:
     def __init__(
         self,
-        width,
+        shape_type,
         position,
         orientation,
         color=(0, 1, 0, 0.5),
+        pybullet_client_id=0,
         **kwargs,
     ):
         """
         Create a cube marker for visualization
 
         Args:
-            width (float): Length of one side of the cube.
+            shape_type: Shape type of the object (e.g. pybullet.GEOM_BOX).
             position: Position (x, y, z)
             orientation: Orientation as quaternion (x, y, z, w)
-            color: Color of the cube as a tuple (r, b, g, q)
+            kwargs: Keyword arguments that are passed to
+                pybullet.createVisualShape.  Use this to specify
+                shape-type-specify parameters like the object size.
         """
-
-        self._kwargs = kwargs
+        self._pybullet_client_id = pybullet_client_id
 
         self.shape_id = pybullet.createVisualShape(
             shapeType=pybullet.GEOM_BOX,
```

Comment:
I think it should be `shapeType=shape_type` here to allow for a shape to be created for any type passed

Reply:
:+1: